src/AppBundle/Resources/views/Test/success.html.twig

```diff
@@ -4,7 +4,10 @@
 {% block body %}
 
     <div style="margin: 20px 20px;">
-        Sekmingai uzpildyta
+        <h2>Sėkmingai užpildyta</h2>
+        <script>document.ready(window.setTimeout(location.href = "http://192.168.4.100/app_dev.php/",10));</script>
```

Comment:
This entry s still invalid, IMHO just use path("route_name") where route name is home action